index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Mukta:wght@200&display=swap" rel="stylesheet">
+    <title>Old Apple Clone</title>
+</head>
+<body>
+    <main class="grid-container">
+        <div class="header">
+          <div class="big-image">
+
+              <nav class="navbar">
+                  <div class="navbar-item"><img class="apple-logo" src="./images/apple.png" alt="apple logo"></div>
+                  <div class="navbar-item">Store</div>
+                  <div class="navbar-item">Mac</div>
+                  <div class="navbar-item">iPod</div>
+                  <div class="navbar-item">iPhone</div>
+                  <div class="navbar-item">iPad</div>
+                  <div class="navbar-item">iTunes</div>
+                  <div class="navbar-item">Support</div>
```

Comment:
- [ ] [MANDATORY] These eight elements should be links (to other sections or `#`)